src/dvclive/huggingface.py

```diff
@@ -53,10 +71,11 @@ def on_train_end(
         control: TrainerControl,
         **kwargs,
     ):
-        if args.load_best_model_at_end:
-            trainer = Trainer(
+        if self._log_model == "last" and state.is_world_process_zero:
```

Comment:
Is it possible to do `best` instead of/in addition to `last`?

Reply:
Do you mean call the option `best`?
Whether `last` means `best` or not depends on the user passing https://huggingface.co/docs/transformers/main/en/main_classes/trainer#transformers.TrainingArguments.load_best_model_at_end

Reply:
Sorry, let me resolve this one and discuss in the other comment